applications/subxt-python.md

```diff
@@ -0,0 +1,196 @@
+# `subxt-python`
+
+- **Team Name:** Philoniare Team
+- **Payment Address:** 14AZk92CKjovm4tDvzE9XsjGz3HNeeV2tmnbYitAqyweWoRp (UDSC)
```

Comment:
Are you aware that we are now paying [30% of each grant in DOT](https://grants.web3.foundation/docs/faq#what-does-it-mean-for-payments-to-be-30-dot)? In this case, DOT and USDC would be sent to the same address. If you prefer, you can add a separate address for the DOT payment.

Reply:
Yep, I'm aware. Thanks for checking

Reply:
Hey @philoniare ,
just wanted to let you know that we really encourage to receive the vested DOT on a native Polkadot address because this way you will able to stake the vested DOT for the whole vesting period. On AssetHub you can't do so. Just a final check for your own benefit.

Reply:
@PieWol updated, thanks for the insightful comment, I wasn't aware